modules/sdk/src/main/scala/org/tessellation/sdk/domain/collateral/LatestBalances.scala

```diff
@@ -0,0 +1,8 @@
+package org.tessellation.sdk.domain.collateral
+
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Balance
+
+trait LatestBalances[F[_]] {
```

Comment:
Do we actually need to track balances? Wouldn't be enough to keep track of a set of adresses that have more than 250k? On every snapshot we would add/remove addresses from that set. It would require less memory because we wouldn't store unnecessary information.

Reply:
@tbekas we store balances cache anyway (either directly in global snapshot or later by applying diffs from light snapshots), so we would just reuse that (read-only). we definitely shouldn't store it twice

Reply:
Makes sense 👍